src/main/java/cms/gov/madie/measure/services/BundleService.java

```diff
@@ -58,19 +67,55 @@ public ResponseEntity<byte[]> exportBundleMeasure(Measure measure, String access
     if (measure == null) {
       return null;
     }
-    if (measure.getMeasureMetaData() != null) {
-      if (CollectionUtils.isEmpty(measure.getMeasureMetaData().getDevelopers())) {
-        throw new InvalidResourceBundleStateException(
-            "Measure", measure.getId(), "since there are no associated developers in metadata.");
-      } else if (measure.getMeasureMetaData().getSteward() == null) {
-        throw new InvalidResourceBundleStateException(
-            "Measure", measure.getId(), "since there is no associated steward in metadata.");
-      } else if (StringUtils.isBlank(measure.getMeasureMetaData().getDescription())) {
+    isMetadataValid(measure);
+    areGroupsValid(measure);
+    try {
+      retrieveElmJson(measure, accessToken);
```

Comment:
Noted `retrieveElmJson` is called under the `draft` branch below. Does it need to be called twice?